src/Core/Scheduler/Providers/3dTiles_Provider.js

```diff
@@ -135,7 +135,7 @@ $3dTiles_Provider.prototype.executeCommand = function executeCommand(command) {
             pnts: this.pntsParse.bind(this),
         };
 
-        return Fetcher.arrayBuffer(url).then((result) => {
+        return Fetcher.arrayBuffer(url, layer.fetchOptions).then((result) => {
```

Comment:
Why did you use sometimes `fetchOptions` and sometimes `textureLoaderOptions` name?

I'd prefer a single name: e.g `fetchOptions` or `networkOptions`